.env.sample

```diff
@@ -1,5 +1,10 @@
 DJANGO_SETTINGS_MODULE=core.settings_prod
 SECRET_KEY=<une chaine de caractère arbitraire ici>
+DATABASE_USER=access4all
+DATABASE_PASSWORD=access4all
+DATABASE_HOST=localhost
+DATABASE_NAME=access4all
+DATABASE_URL=postgres://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}/${DATABASE_NAME}
```

Comment:
Yes, env vars other than DATABASE_URL won't ever reach production, and DATABASE_URL will be fully overwrited. I agree that it may be confusing. Other solution could be to duplicate the data between .env and docker compose, because the docker image need separated vars.